### ISSUE TITLE
Merge Habits Branch to Main Branch to Prod

### ISSUE THREAD

**Issue by TMJSMC53** [TMJSMC53/holistic-health-app#43]
# User Story

User will be able to use the built-in habits

# Potential Solution

After #42 , the Habits branch will be merged into the Main branch

### Frontend
- [x] N/A

### Model
- [x] N/A

### Route
- [x] N/A

### Controller
- [x] N/A

# Acceptence Criteria
- [x] Users will be able to use the built-in habits
- [x] Habits branch will be merged into the Main branch and then deleted
- [x] Push Main changes to Prod
